gradle.properties

```
@@ -21,15 +21,15 @@ testngVersion=7.4.0
 jacocoVersion=0.8.6
 wso2OrbitAntlrVersion=4.5.1.wso2v1
 ballerinaGradlePluginVersion=1.0.0
-bouncycastleVersion=1.69
+bouncycastleVersion=1.74
 wso2CommonsPoolVersion=1.5.6.wso2v1
 
 #stdlib dependencies
 stdlibConstraintVersion=1.0.1
 stdlibCryptoVersion=2.3.0
 stdlibIoVersion=1.3.1
 stdlibLogVersion=2.5.0
-stdlibHttpVersion=2.5.3
+stdlibHttpVersion=2.5.4-20231013-152300-784fd78
```

Update http version after release
```suggestion
stdlibHttpVersion=2.5.4
```